core/modules/update/update.admin.inc

```diff
@@ -35,6 +35,12 @@ function update_settings($form, &$form_state) {
     '#title' => t('Also check for updates of disabled modules, themes, and layouts'),
     '#default_value' => $config->get('update_disabled_extensions'),
   );
+  $form['check']['update_report_usage'] = array(
+    '#type' => 'checkbox',
+    '#title' => t('Allow this site to be included in usage statistics'),
```

Comment:
"Allow this site to be included..." make this sound like the site name/ip is being tracked.

```suggestion
    '#title' => t('Send anonymous usage statistics'),
```